### ISSUE TITLE
Mysterious `settracker()` issue

### ISSUE THREAD

**Issue by phylll** [michael-buschbeck/mychs-macro-magic#235]
I need your help again. Same issue we discussed in late May on Discord: how do we get the turn number to show up in the combat turn tracker?

I am unable to reproduce the problem in the test game. In the main game, however, it is very reproducible between different maps with down to a single character token present. The present code of my `startTracker` script (stored in an `MacroSheet` ability) is this:

```
!mmm script
!mmm   
!mmm   for token in gettokens() where not isdenied(token.character_name)
!mmm     set trackerData = trackerData, { token: token.token_id, value: token.Gw + token.ItemBGewandheit + token.modGw + token.gRusGW }
!mmm   end for
!mmm do chat(count(trackerData) & " Einträge")
!mmm   set trackerData = { title: "Runde", value: "1", formula: "+1" }, trackerData order (...left.value > ...right.value)
!mmm do chat("hinterher: " & count(trackerData) & " Einträge")
!mmm for i in trackerData
!mmm   do chat(i.title & ": " & i.token.token_name & "(" &  i.token & ") - " & i.value & " - " & i.formula )
!mmm end for
!mmm   do settracker(trackerData)
!mmm   do showtracker(true)
!mmm
!mmm end script
```

The output that produces is, in the simplest case of a new, empty map with only one token, this:

```
11:15PM
Wulfric MacConuilh: 1 Einträge
hinterher: 2 Einträge
Runde: () - 1 - +1
: Yorric MacRathgar(-O7kB3k1507-Q7gxlqmH) - 64 -
```

So the token is being found by `gettokens()` and added to the `trackerData` struct. The turn number entry is also added to `trackerData`, without a `token` attribute as you suggested in May (doesn't seem to matter either way in my tests now in the main game). What is being handed over to `settracker()` looks fine. And still, the tracker appears with only Yorric being listed, and no turn number entry.

Could you perhaps investigate a bit more, with your GM privileges and perhaps under the MMM hood?

**Comment by michael-buschbeck** [comment]
The main game is still using an outdated MMM version (1.36.1) that doesn't have the latest fix (1.36.2). I'll upgrade it.

**Comment by michael-buschbeck** [comment]
Upgraded the main game to 1.36.2, and I can confirm that the test command above works as expected now.
